src/Avalonia.Controls/Platform/Screen.cs

```diff
@@ -1,21 +1,42 @@
 ﻿namespace Avalonia.Platform
 {
+    /// <summary>
+    /// Represents a single display screen.
+    /// </summary>
     public class Screen
     {
+        /// <summary>
+        /// Gets the pixel density of the screen.
+        /// This is a scaling factor so multiply by 100 to get a percentage.
+        /// </summary>
+        /// <remarks>
+        /// Both X and Y density are assumed uniform.
+        /// </remarks>
         public double PixelDensity { get; }
 
+        /// <summary>
+        /// Gets the overall pixel-size of the screen.
+        /// This generally is the raw pixel counts in both the X and Y direction.
+        /// </summary>
         public PixelRect Bounds { get; }
 
+        /// <summary>
+        /// Gets the actual working-area pixel-size of the screen.
+        /// This may be smaller to account for notches and other block-out areas.
```

Comment:
Yea, I wasn't sure other than a notch what else this might be adjusting for. Forgot about how macOS does the taskbar though. I've included your comment and made it a remark now.

Some Bounds property comments were moved to remarks for uniformity.